src/mjolnir/restrictionbuilder.cc

```diff
@@ -30,6 +31,43 @@ struct EdgeId {
   GraphId graph_id;
 };
 
+GraphId GetOpposingEdge(GraphReader& reader,
```

Comment:
Copied it from the shortcut code https://github.com/valhalla/valhalla/blob/da75fe9b757123faf172511c25b54570f666d09f/src/mjolnir/shortcutbuilder.cc#L97